i18n.ts

```diff
@@ -12,13 +12,13 @@ const md = MarkdownIt({
 function loadLocaleMessages(): LocaleMessages {
   // File containing data common to ALL languages
   const allLangDataFile = 'all_lang.json'
-  const locales = require.context('./langDir', true, /[A-Za-z0-9-_,\s]+\.json$/i)
+  const locales = require.context('./langDir', true, /[A-Za-z0-9-_,\s]{1,8}\.json$/i)
   const messages: LocaleMessages = {}
   locales.keys().forEach(key => {
     if (key === allLangDataFile) {
       return
     }
-    const matched = key.match(/([A-Za-z0-9-_]+)\./i)
+    const matched = key.match(/([A-Za-z0-9-_]{1,8})\./i)
```

Comment:
why 8?

Reply:
since the longest name of a json file in langDir is all_lang, which also happens to have 8 characters

Reply:
having + in this case would be abused in NFA (non deterministic finite automation) so i tried to only evaluate the number of necessary characters